pkg/model/bootstrapscript.go

```diff
@@ -94,6 +96,13 @@ func (b *BootstrapScript) ResourceNodeUp(ig *kops.InstanceGroup, cs *kops.Cluste
 			if os.Getenv("AWS_REGION") != "" {
 				return fmt.Sprintf("export AWS_REGION=%s\n",
 					os.Getenv("AWS_REGION"))
+			} else {
+				sess := session.Must(session.NewSession(aws.NewConfig().
+					WithMaxRetries(3),
+				))
+				if *sess.Config.Region != "" {
+					return fmt.Sprintf("export AWS_REGION=%s\n", *sess.Config.Region)
```

Comment:
So this impacts the nodeup code I think, not the kops aws code.  What exactly are you trying to fix??